SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LegacyLoadBalancingStrategy.cpp

```diff
@@ -66,6 +69,45 @@ class FCustomWorkerAssignmentStorage : public TLBDataStorage<LegacyLB_CustomWork
 {
 };
 
+class DbgPositionStorage : public FSpatialPositionStorage
```

Comment:
This is only used for the performance benchmark right?

Reply:
Yep